wcivf/apps/elections/templates/elections/includes/_post_meta_title.html

```diff
@@ -1 +1,5 @@
-{{ object.election.name }}: The {{ person_posts|length }} candidates in {{ object.post.label }}
\ No newline at end of file
+{% if object.election.current_or_future_elections %}
+{{ object.election.name }}: The {{ postelection.people|length }} candidates in {{ object.post.label }}
+{% else %}
+{{ object.election.name }}: The official candidate list has not yet been published.
```

Comment:
Sorry to dive in when this is still WIP, but I thought a comment on this might be useful now:

It's not quite true that the states are "have candidates" or "SOPN not published", although they are the most common states we're likely to see.

On WCIVF we import candidates before they've been "verified" (e.g, the SOPN is out, two people have compared our list to the SOPN). This means that we could have a single candidate on the site months before the SOPN. This code deals with that case well.

However, we can also have a state where the SOPN is out but we've not managed to add any candidates for it yet. It can take some days for us to get to a given area, this year especially.

There are a couple of options here:

1. We get clever with another condition (Using https://github.com/DemocracyClub/uk-election-timetables as we do [elsewhere in this project](https://github.com/DemocracyClub/WhoCanIVoteFor/blob/fa251d0910e228791b31309ad3025bd7ff2f2d30/wcivf/apps/elections/helpers.py#L98))
2. We make the above line more generic, something like "no candidates known yet" - maybe ask @pmk01 for some ideas here.

As this is just a page title, I think option 2 is the better one here, but happy for either if you think 1is better.

Reply:
No problem! With #1, are you suggesting a third condition? In the mean time, I've pushed the change as suggested in #2.

Reply:
Yeah, if we were to implement #1, it would need to deal with another condition. You can see how we do that on the ballot card here: https://github.com/DemocracyClub/WhoCanIVoteFor/blob/master/wcivf/apps/elections/templates/elections/includes/_single_ballot.html#L100

I think going down that route is overkill for a `<title>` though, especially as we have it on the page already